Source/Core/Core/AchievementManager.h

```diff
@@ -35,6 +38,18 @@ class AchievementManager
   void Logout();
   void Shutdown();
 
+  void EnableDLL(bool enable);
+  bool IsDLLEnabled();
+
+#ifdef ENABLE_RAINTEGRATION
+  void ReinstallMemoryBanks();
+  void MainWindowChanged(void* new_handle);
+  void GameChanged(bool isWii);
+  void RAIDoFrame();
+  std::vector<std::tuple<int, std::string, bool>> GetMenuItems();
```

Comment:
Maybe declare this return type as an alias like:

```cpp
using DescriptiveName = std::vector<std::tuple<int, std::string, bool>>;
```

and explain what the entries mean, or make a struct like:

```cpp
// Acts as a translation struct to keep the RA equivalent private to the implementation
struct MenuItem {
  // These two allow emplace_back to construct in place instead of copying
  MenuItem() = default;
  MenuItem(int id, std::string name, bool is_checked) : id{id}, name{std::move(name)}, is_checked{is_checked} {}

  int id = 0;
  std::string name;
  bool is_checked = false;
};
```

and then use it like:

```cpp
std::vector<MenuItem> GetMenuItems();
```

so people don't need to read the implementation to understand what's being returned here.